packages/Radio/src/Radio.styles.js

```diff
@@ -127,7 +118,7 @@ export const Radio = styled.div(
         ${styles[size].radioStyles};
         ${z(1)};
         background: ${tokens.color.white};
-        border: 2px solid ${tokens.border.color};
+        border: 2px solid ${tokens.border.hoverColor};
```

Comment:
small one, since the style is not for a hovered element, I think the token `blackLighten40` makes more sense